src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_ploidy.py

```diff
@@ -121,6 +123,8 @@ def __call__(self):
             log_q_ploidy_jk = self.ploidy_workspace.log_q_ploidy_sjk.get_value(borrow=True)[si, :, :]
             for j in range(self.ploidy_workspace.num_contigs):
                 ploidy_j[j], ploidy_genotyping_quality_j[j] = model_commons.perform_genotyping(log_q_ploidy_jk[j, :])
+                if ploidy_genotyping_quality_j[j] < self.ploidy_gq_filter:
+                    ploidy_j[j] = np.argmax(self.ploidy_config.contig_ploidy_prior_map.get(j))
```

Comment:
I think an info message should be emitted with this

Reply:
I was tinkering with this to try to fix some of the aneuploid 1000G calls, but the WGS data agrees that they're at least mosaic aneuploid.  I'll take it out.